src/main/java/net/minecraftforge/common/data/ForgeItemTagsProvider.java

```diff
@@ -111,9 +113,14 @@ public void addTags()
         copy(Tags.Blocks.ORES_QUARTZ, Tags.Items.ORES_QUARTZ);
         copy(Tags.Blocks.ORES_REDSTONE, Tags.Items.ORES_REDSTONE);
         copy(Tags.Blocks.ORES_NETHERITE_SCRAP, Tags.Items.ORES_NETHERITE_SCRAP);
+        copy(Tags.Blocks.ORES_COPPER, Tags.Items.ORES_COPPER);
         tag(Tags.Items.RODS).addTags(Tags.Items.RODS_BLAZE, Tags.Items.RODS_WOODEN);
         tag(Tags.Items.RODS_BLAZE).add(Items.BLAZE_ROD);
         tag(Tags.Items.RODS_WOODEN).add(Items.STICK);
+        tag(Tags.Items.RAW_ORES).addTags();
```

Comment:
This should include `RAW_ORES_GOLD`, `RAW_ORES_IRON`, and `RAW_ORES_COPPER` I guess.

Reply:
Jup, missed that 🤦‍♂️